tester_app/client/webpack.client.base.config.js

```diff
@@ -0,0 +1,58 @@
+// Common client-side webpack configuration used by webpack.hot.config and webpack.rails.config.
+
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+
+  // the project dir
+  context: __dirname,
+  entry: {
+
+    // See use of 'vendor' in the CommonsChunkPlugin inclusion below.
+    vendor: [
+      'babel-core/polyfill',
+      'jquery',
+      'jquery-ujs',
+      'react',
+      'react-dom',
+    ],
```

Comment:
@justin808 That was my understanding as well. A lot of people think of it as the standard library that *should* have come with JavaScript and always just include it.